thirdparty/README.md

```diff
@@ -439,6 +439,20 @@ Files extracted from upstream source:
 - License.txt
 
 
+## Rvo2-3D
```

Comment:
Use the same name as the folder, so `rvo2`.

Reply:
Done